SConstruct

```diff
@@ -244,6 +244,7 @@ condition_map = {
     'POWERPC_HOST' : False,
     'WINDOWS_HOST' : env['PLATFORM'] == 'win32',
     'X86_HOST' : True,
+    'MIPS64EL_HOST' : True,
```

Comment:
Is it necessary to set this to `True`? Our automated testing couldn't compile on Windows and I suspect that this is why.

Reply:
Sorry, not here